Scripts/Runtime/Mathematics/MathExtension.cs

```diff
@@ -0,0 +1,99 @@
+using System.Runtime.CompilerServices;
+using Unity.Mathematics;
+
+namespace Anvil.Unity.DOTS.Mathematics
+{
+    /// <summary>
+    /// A collection of extension methods for working with types under the <see cref="Unity.Mathematics"/> namespace.
+    /// (float3, int2, etc..)
+    /// </summary>
+    public static class MathExtension
```

Comment:
Same questions as PR in Unity Core about `MathematicsExtension` instead of `MathExtension`